docs/docs/overview/why_secretless.md

```diff
@@ -1,52 +1,7 @@
 ---
 title: Why Secretless?
 id: docs
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/overview/why_secretless.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
```

Comment:
this content was moved here: https://docs.secretless.io/Latest/en/Content/Overview/overview.htm